### ISSUE TITLE
Download all icons from website

### ISSUE THREAD

**Issue by sn0wyfall** [elrumo/macOS_Big_Sur_icons_replacements#1034]
It it possible to download all of the icons from the website? Previously I'd just clone the github repo but it seems like the newer icons aren't being added to github anymore.

**Comment by elrumo** [comment]
Yes, for the last couple of months the icons are not being hosted on Github anymore as I build a backend system that automates the creation of .icns files, that way you can submit PNGs directly to the site. I'll need to think what the best way to let everyone have access to all the icons.

**Comment by sn0wyfall** [comment]
Thanks, or maybe give us a way to just via alphabet. It's so annoying when I've scrolled past 2000 icons and I accidentally refresh the page, and I need to scroll down 2000 icons again, especially with the way the icons load in batches
